src/pages/index.js

```diff
@@ -233,8 +233,8 @@ function handleEditFormSubmit(evt) {
       disableButton(profileSubmitButton, settings);
       closeModal(editProfileModal);
     })
-    .catch(console.error)
-    .finally((submitButton.textContent = "Save"));
+    .catch(() => console.error)
```

Comment:
` .catch(console.error);` should be used instead of `.catch(() => console.error)`.  


It should be either `.catch(() => console.error());` or ` .catch(console.error);`, but not  `.catch(() => console.error)` because the `console.error` is not called inside the arrow function, so it doen't work


**Please, correct it everywhere in the code**